### ISSUE TITLE
Screen readers are not supported

### ISSUE THREAD

**Issue by sonyashnyk** [telegramdesktop/tdesktop#24265]
### Steps to reproduce

I use Telegram and NVDA.

### Expected behaviour

If a screen reader is used, then when exploring the interface with the mouse or system focus, all interface elements should be spoken.

### Actual behaviour

The interface is not available at all.

### Operating system

Win 10 1809 17763.2686

### Version of Telegram Desktop

3.6.1 x64

### Installation source

Static binary from official website

**Comment by ilya-fedin** [comment]
#476
